srcpkgs/OpenRCT2/template

```diff
@@ -2,11 +2,11 @@
 # based on https://raw.githubusercontent.com/AluisioASG/void-packages/openrct2/srcpkgs/OpenRCT2/template
 # and https://github.com/void-linux/void-packages/issues/1014#issuecomment-417372421
 pkgname=OpenRCT2
-version=0.4.13
+version=0.4.14
 revision=1
 # versions pulled from https://raw.githubusercontent.com/OpenRCT2/OpenRCT2/v${version}/CMakeLists.txt
 _objects_version=1.4.7
-_titles_version=0.4.6
+_titles_version=0.4.14
```

Comment:
And since I checked out of curiosity, no, we haven't been missing/forgetting updates to this dependency along the way, upstream truly did jump 8 .z releases in one hit (makes sense, title sequences only need re-released when re-animated/scripted).

https://github.com/OpenRCT2/title-sequences/releases